src/test/java/fr/insee/pearljam/integration/surveyunit/SurveyUnitIT.java

```diff
@@ -231,19 +231,20 @@ void testGetSurveyUnitDetail() throws Exception {
 				   },
 				   "communicationTemplates":[
 				      {
-				         "id":1,
+				         "meshuggahId":"mesh1",
 				         "medium":"EMAIL",
 				         "type":"REMINDER"
 				      },
 				      {
-				         "id":2,
+				         "meshuggahId":"mesh2",
 				         "medium":"LETTER",
 				         "type":"NOTICE"
 				      }
 				   ],
 				   "communicationRequests":[
 				      {
-				         "communicationTemplateId":1,
+				         "campaignId":"SIMPSONS2020X00",
```

Comment:
warining: same logic here :
meshuggahId should be duplicated as communicationTemplateId